public/api-merged.json

```diff
@@ -7677,40 +7677,6 @@
         }
       }
     },
-    "/playlists/{uid}/dashboards": {
```

Comment:
we have to remove it also from 
[public/openapi3.json](https://github.com/grafana/grafana/blob/d24dc7f1560929b1df5de9f644df49fa0438b616/public/openapi3.json#L19670)

they can be re-created both by using:
https://github.com/grafana/grafana/blob/d24dc7f1560929b1df5de9f644df49fa0438b616/pkg/api/README.md?plain=1#L71-L82